src/mpyl/utilities/jenkins/runner.py

```diff
@@ -89,52 +95,68 @@ def await_parameter_build(self, build_job: Job):
 
     @staticmethod
     def to_icon(build_result: Build) -> str:
-        return '✅ ' if build_result.is_good() else '❌ '
+        status = build_result.get_status()
+        if status == STATUS_SUCCESS:
+            return '✅ '
+        if status == STATUS_ABORTED:
+            return '🚫 '
+        return '❌ '
 
-    def follow_logs(self, job: Job, build_number: int, duration_estimation: int):
-        self.status.update("Waiting for logs....")
-        while job.get_last_buildnumber() != build_number:
-            time.sleep(1)
-        self.status.stop()
+    def follow_logs(self, job: Job, build_number: int, duration_estimation: int, verbose: bool = False):
 
         build_to_follow: Build = job.get_build(build_number)
         self.status.console.log(f'{build_to_follow} {self.pipeline.build_location()}')
 
-        with Progress(console=self.status.console) as progress:
-            build_task = progress.add_task("", total=duration_estimation, visible=duration_estimation > 0)
+        self._stream_logs(build_to_follow, duration_estimation, verbose)
+
+        build_to_follow.block_until_complete()
+        finished_build = job.get_last_build()
+        self.status.console.log(
+            f'[link={finished_build.get_build_url()}][i]Build[/link] for {self.pipeline.human_readable()} '
+            f'ended with outcome {self.to_icon(finished_build)}', markup=True)
+        self.status.console.log()
+
+        os.system('afplay /System/Library/Sounds/' + ('Glass.aiff' if finished_build.is_good() else 'Sosumi.aiff'))
+        sys.exit()
```

Comment:
So we expect mpyl to be run on a mac?

Reply:
ah, yeah, good one. We should add a condition here.